runtime/src/main/java/org/corfudb/runtime/Backup.java

```diff
@@ -156,33 +171,40 @@ private void backup() throws IOException {
      * If the log is trimmed at timestamp, the backupTable will fail.
      * If the table has no data to be backed up, it will create a file with empty contents.
      *
-     * @param fileName   - the name of the backup file
+     * @param filePath   - the path of the backup file
      * @param uuid       - the uuid of the table which is being backed up
      * @throws IOException
      */
-    private void backupTable(String fileName, UUID uuid) throws IOException {
-        try (FileOutputStream fileOutput = new FileOutputStream(fileName)) {
+    private void backupTable(Path filePath, UUID uuid) throws IOException {
+        int numberOfEntries;
+        long startTime = System.currentTimeMillis();
+
+        try (FileOutputStream fileOutput = new FileOutputStream(filePath.toString())) {
             StreamOptions options = StreamOptions.builder()
                     .ignoreTrimmed(false)
                     .cacheEntries(false)
                     .build();
             Stream<OpaqueEntry> stream = (new OpaqueStream(runtime.getStreamsView().get(uuid, options))).streamUpTo(timestamp);
 
-            writeTableToFile(fileOutput, stream, uuid);
+            numberOfEntries = writeTableToFile(fileOutput, stream, uuid);
         } catch (IOException e) {
-            log.error("failed to back up table {} to file {}", uuid, fileName);
+            log.error("failed to back up table {} to file {}", uuid, filePath.toString());
             throw e;
         } catch (TrimmedException e) {
             log.error("failed to back up tables as log was trimmed after back up starts.");
             throw e;
         }
 
-        log.info("{} table is backed up and stored to temp file {}", uuid, fileName);
+        long elapsedTime = System.currentTimeMillis() - startTime;
+
+        log.info("{} entries (elapsed time {}ms) saved to temp file {}", numberOfEntries, elapsedTime, filePath.toString());
     }
 
-    private void writeTableToFile(FileOutputStream fileOutput, Stream<OpaqueEntry> stream, UUID uuid) throws IOException {
+    private int writeTableToFile(FileOutputStream fileOutput, Stream<OpaqueEntry> stream, UUID uuid) throws IOException {
```

Comment:
Would be great to create a class with some stats and return it instead of just a number of tables
For instance
```
class WriteTableStats {
    int tablesNum;
    long totalBytesWritten;
    List<UUID> tableNames
}
```

Reply:
This function only writes one table, so `tableNums` and `tableNames` are not really meaningful here. It's a good idea to wrap `numOfEntries` and `totablBytesWritten` in a class. For the current requirements we have received it is enough.

Reply:
So, it will return 1 always? Stil why not return more info instead of meaningless int number? Next time when someone will need that info, that developer will need to add that info anyways. And this info is useful even for backup right now, why wouldn't like to provide that info?

Reply:
I added a `BackupTableStats` class which reports `numOfEntries` and `tableSize` in bytes.